### PR TITLE
docs: improve external-approval

### PR DESCRIPTION
**Pull request by RainbowDashy**
Add more details on how to enable.

### CODE REVIEW

File: content/docs/administration/webhook-integration/external-approval.md [hunk 1]
```diff
@@ -22,7 +22,13 @@ Fill in App Name and Description such as Bytebase Approval.
 
 ![feishu app creation page](/static/docs/external-approval-feishu-create.webp)
 
-### Step 2 - Grant permissions
+### Step 2 - Upload Icon
+
+In the General info section of Credentials & Basic Info page, add [Bytebase icon](https://www.bytebase.com/brand).
```
Comment (tianzhou) [inline content/docs/administration/webhook-integration/external-approval.md:27]: We can inline icon image here /assets/logo-icon.svg

File: content/docs/administration/webhook-integration/external-approval.md [hunk 2]
```diff
@@ -32,33 +38,43 @@ In the Manage scopes section of Permissions & Scopes page, add the following sco
...
+
 Submit for release.
 
 ![feishu app release submission page](/static/docs/external-approval-feishu-enable-3.webp)
 
-Ask your organization administrator to approve the custom application in [Workspace / App review](https://feishu.cn/admin/appCenter/audit).
+Ask your **organization administrator** to approve the custom application in [Workspace / App review](https://feishu.cn/admin/appCenter/audit).
 
-### Step 4 - Obtain credentials
+Move to the next step when you see the app status is **"Enabled"**.
```
Comment (tianzhou) [inline content/docs/administration/webhook-integration/external-approval.md:61]: leave a new line between text and image

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Review by candy2255 — APPROVED** [review]
LGTM

File: content/docs/administration/webhook-integration/external-approval.md (hunk 1 above)
Comment (RainbowDashy) [reply content/docs/administration/webhook-integration/external-approval.md:29]: We can't access files in /assets so we have to copy one to /static.

File: content/docs/administration/webhook-integration/external-approval.md [hunk 3]
```diff
@@ -22,7 +22,15 @@ Fill in App Name and Description such as Bytebase Approval.
 
 ![feishu app creation page](/static/docs/external-approval-feishu-create.webp)
 
-### Step 2 - Grant permissions
+### Step 2 - Upload Icon
+
+![Bytebase logo icon](/static/docs/logo-icon.svg)
+
+In the General info section of Credentials & Basic Info page, add [Bytebase icon](https://www.bytebase.com/brand).
```
Comment (tianzhou) [inline content/docs/administration/webhook-integration/external-approval.md:29]:
```suggestion
In the General info section of Credentials & Basic Info page, add the above Bytebase icon.
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**